keyboards/handwired/dactyl_manuform_pi_pico/readme.md

```diff
@@ -0,0 +1,32 @@
+# dactyl_manuform_pi_pico
+
+![dactyl_manuform_pi_pico](https://github.com/GustawXYZ/dactyl_manuform_pi_pico/blob/main/images/overview.jpg?raw=true)
```

Comment:
Image should be rehosted.

Reply:
Thanks, what would be a good place to put it?

My guess was that this repo isn't the place for it, as it would bloat the size. But I don't have any better ideas other than in this complimentary guide-repo

Reply:
Imgur is the default recommendation.  Though, make sure you have an account and upload it via that account, to make sure it doesn't get randomly pruned.

Reply:
Fixed, thank you for the review, I hope that it's all good now :D